### ISSUE TITLE
[heft] Update node-sass dependency to 5.0.0 in order to fix builds with Node 15

### ISSUE THREAD

**Issue by Niek** [microsoft/rushstack#2490]
Installing heft on Linux with Node 15 doesn't work - I get the following errors:

```bash
.../node_modules/node-sass install$ node scripts/install.js
../node_modules/node-sass install: Downloading binary from https://github.com/sass/node-sass/releases/download/v4.14.1/linux-x64-88_binding.node
.../node_modules/node-sass install: Cannot download "https://github.com/sass/node-sass/releases/download/v4.14.1/linux-x64-88_binding.node": 
.../node_modules/node-sass install: HTTP error 404 Not Found
.../node_modules/node-sass install: Hint: If github.com is not accessible in your location
.../node_modules/node-sass install:       try setting a proxy via HTTP_PROXY, e.g. 
.../node_modules/node-sass install:       export HTTP_PROXY=http://example.com:1234
.../node_modules/node-sass install: or configure npm proxy via
.../node_modules/node-sass install:       npm config set proxy http://example.com:8080
.../node_modules/node-sass install: Done
```

This can be fixed by bumping the dependency of node-sass to 5.0.0, which has releases for Node 15.

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@rushstack/heft` version? | 0.24.2 |
| Operating system? | Linux |
| Would you consider contributing a PR? | Yes |
| Node.js version (`node -v`)? | 15.8.0 |

**Comment by halfnibble** [comment]
According to [this](https://www.npmjs.com/package/node-sass). Bumping to version 5.0.0 would lose support for:
Node 8
Node 11
Node 13

I don't think we officially support those, so is that alright @octogonz or @iclanton?

**Comment by iclanton** [comment]
Rush supports Node 8, but I don't think anything else needs to.

We don't officially support Node 11 or 13.

**Comment by octogonz** [comment]
Sounds good to me.  @Niek do you want to make a PR?

**Comment by octogonz** [comment]
@halfnibble said it will be merged soon

**Comment by Niek** [comment]
I see it's fixed in heft@0.25.1, thanks a lot!
